Sources_v3/Workers/EventSender.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+extension TimeInterval {
+    /// The number of seconds from the last `typingStart` event until the `typingStop` event is automatically sent.
+    static let startTypingEventTimeout: TimeInterval = 5
+    
+    /// If user is still typing, resend the `typingStart` event after this time interval.
+    static let startTypingResendInterval: TimeInterval = 20
+}
+
+/// Sends events.
+class EventSender<ExtraData: ExtraDataTypes>: Worker {
```

Comment:
Are you sure we need a special worker for this? Since typing events make sense only for a channel, what's wrong with adding this to `ChannelUpdater`?

If you want to keep it separate (I'm ok with that), why not name it `TypingEventsSender` then?

Reply:
I thought about to add it to `ChannelUpdater`, but it's not really related to channel updating and it has additional properties that's good to separate. I think it's good to name it as a general event sender, because basically we can send any event to our backend.